Libraries/Lists/__tests__/__snapshots__/VirtualizedList-test.js.snap

```diff
@@ -728,7 +862,7 @@ exports[`VirtualizedList keeps sticky headers realized after scrolled out of vie
     <View
       style={null}
     >
-      <item
+      <MockCellItem
```

Comment:
This snapshot was described as:
>  // Scroll to the bottom 50 dip (last five items) of the content. Expect the
    // last five items to be rendered, along with every sticky header above,
    // even though they are out of the viewport window in layout coordinates.
    // This is because they will remain rendered even once scrolled-past in
    // layout space.

I see more than the last 5 items rendered here -- am I misunderstanding?

Reply:
You're right. The description here isn't quite accurate. I can update it.

We always keep the initially rendered items present rendered, for the scroll to top optimization. Since initialNumToRender is set to 1, we keep the top cell rendered. `windowSize` is set to 1, instead of the default 21, which should theoretically mean "just keep the stuff in viewport rendered". This would mean five items, and I've observed that is what we do end up seeing when viewport is at the top, or some other positions. I've noticed we sometimes render more than that though, which I think may be related to either rounding, or directional render logic, that I have not explored. That is I think why we see more than 5 items at the end, even though the contract would be that `windowSize` of 1 would mean just the items in viewport.

Now that I am looking closer at this, we have a 50-height spacer where some sticky headers should be. We do specifically keep MockCellItem 6 rendered, even though there are two spacers after, but that implies we are only keeping the most recent sticky header above viewport realized, instead of all of them.

Reply:
Updated the description to better match the current behavior